README.md

```diff
@@ -119,3 +120,29 @@ to_field 'agg_has_view' do |_record, accumulator, context|
   accumulator << transform_values(context, 'wr_id' => extract_marc('856u', first: true))
 end
 ```
+
+### JSON
+
+#### Generic JSON
+
+##### Example Source Data
+
+```
+{
+	"title": "JSON Item Title",
+	"thumbnail": {
+		"resource": "http://www.example.com/thumbs/12345.jpg"
+	}
+}
+```
+
+##### Example Configuration
+
+```
+to_field('cho_title'), extract_json('.title', )
+to_field 'agg_is_shown_at' do |_record, accumulator, context|
+  accumulator << transform_values(context,
+                                  'wr_id' => [extract_json('.thumbnail.resource')
```

Comment:
missing a `]` here I think.

Reply:
Hah, I copied it from the XML example above.